src/components/results/generic/index.js

```diff
@@ -0,0 +1,48 @@
+import {useState, useEffect} from "react";
+import useGraphql from "lib/hooks/use-graphql";
+import useHttp from "lib/hooks/use-http";
+import useResults from "lib/hooks/use-results";
+import Breakdown from "./breakdown";
+import Header from "./header";
+import ResultActions from "./result_actions";
+import Score from "./score";
+import style from "./style.scss";
+
+export default function Generic() {
+  const [result, setResult] = useState(null);
+  const assessment = useResults({surveyType: "generic"});
+  const profile = result ? result.profile : {};
+  const assessmentResult = result ? result.assessment : {};
+
+  const graphQL = useGraphql();
+  const http = useHttp();
+
+  useEffect(() => {
+    if(assessment === null) return;
+    const query = graphQL.generic.result;
+    const variables = {assessmentID: assessment.id};
+
+    http.post(graphQL.generic.path, {query, variables}).then(({data, errors}) => {
```

Comment:
This should come back naturally with our state management in the useResults hook. Usually one endpoint can return the questions/slides if incomplete and results if complete. That's what would be preferred, but if that doesn't work, we could just have it make a second query [here](https://github.com/traitify/traitify-widgets/blob/TD-2500-render-results/src/lib/recoil/assessment.js#L105) if the questions are complete.

I think we'd want it to return a structure that incorporates them both